cves/CVE-2009-1890.yml

```diff
@@ -3,22 +3,28 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+
+CWE: 189
+
 CVSS: AV:N/AC:M/Au:N/C:N/I:N/A:C
+
 curated_instructions: |
   If you are manually editing this file, then you are "curating" it. Set the
   entry below to "true" as soon as you start. This will enable additional
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+
+curated: true 
+
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
   source for this is Chrome's Stable Release Channel
   (https://chromereleases.googleblog.com/).
   Please enter your date in YYYY-MM-DD format.
-announced: 2009-07-05T16:30Z
+announced: 2009-06-02 (CVE Log Date)
```

Comment:
Please take off "(CVE Log Date)," this makes it easier on us when it parses

Reply:
Done!